dev/validate_bom__config.py

```diff
@@ -1003,7 +1003,31 @@ class SecurityConfigurator(Configurator):
   pass
 
 
+class SpinnakerConfigurator(object):
+  """Controls spinnaker-local overrides."""
+
+  def init_argument_parser(self, parser):
+    """Implements interface."""
+    pass
+
+  def validate_options(self, options):
+    """Implements interface."""
+    pass
+
+  def add_config(self, options, script):
+    """Implements interface."""
+    script.append('mkdir -p  ~/.hal/default/profiles')
+    script.append('echo "management.security.enabled: false"'
```

Comment:
Discussed offline - decided that a flag was not needed until a use case for it is found. Eric is working on an implementation that doesn't need to scrape `/resolvedEnv`, which is why we needed to disable management endpoint security to begin with.